ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitor.java

```diff
@@ -932,7 +933,7 @@ private CastVisitor() {
         final int scale = decimal.getScale();
         exprStr = String.format("DecimalUtil.format(%d, %d, %s)", precision, scale, expr.getLeft());
       } else {
-        exprStr = "String.valueOf(" + expr.getLeft() + ")";
+        exprStr = "Objects.toString(" + expr.getLeft() + ", null)";
```

Comment:
looking at the `plan.json` that was generated from the test, I'm not sure this is backwards compatible - unless I'm missing something, we don't add the generated java code to the plan that we enqueue (cc @rodesai to comment on whether or not including that might be a good idea). That would mean that we would be changing any existing query to have this behavior.

we should consider whether this backwards incompatible change is acceptable or if we should guard this with a config that defaults to disabled, but any new query would have it enabled